feature_engine/selection/shuffle_features.py

```diff
@@ -169,6 +170,8 @@ def fit(self, X: pd.DataFrame, y: pd.Series):
 
         if isinstance(y, pd.Series):
             y = y.reset_index(drop=True)
+        else:
```

Comment:
if y is a pd.Series already, why do we need to re-assign?

Reply:
I must say that I am a bit puzzled by this one. My reasoning was: if there is a condition on y being and pd.Series then y could in some cases also not be a Series. Because I make a selection using `.iloc` I though of forcing y to be a pd.Series.
However when I do that it the unit tests fails and throw an error I cannot reproduce on my development notebook. I now removed it but I must say I am not 100% confident this is the right choice.